rust/worker/src/blockstore/arrow/provider.rs

```diff
@@ -169,8 +169,8 @@ impl BlockManager {
                         let res = bytes.read_to_end(&mut buf).await;
                         match res {
                             Ok(_) => {}
-                            Err(_) => {
-                                // TODO: log error
+                            Err(e) => {
```

Comment:
I think we should leave this TODO in some fashion, this should return a Result<> IMO